joint_state_topic: "/joint_states"
target_position_topic: "/target_position"
gripper_state_topic: "/gripper_state"
cube_pose_topic: "/cube_pose"
pick_toggle_topic: "/pick_toggle"
ee_control_service_name: "/ee_control_service"